calblueprint/siren-web
# Bry/documents view

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] display image in external link
 - [x] get path to access docs in firebase
 - [ ] match with how Greg displays case in Client View
 - [x] display every case's required docs & shows if they are uploaded or not

## How can the reviewer test your code? 👩‍💻
_example: run `npm run dev` and click on a Client row. Then go into the Documents box and click on a link. It should open a new tab and display the image.
<img width="458" alt="Screen Shot 2022-05-09 at 3 35 09 PM" src="https://user-images.githubusercontent.com/60284767/167509635-2ce55df8-6983-4f46-8e17-9de43d9c871c.png">

🧜‍♀️ cc: @shannonbonet @phoebeli23
